src/libs/google-drive-uploader.js

```diff
@@ -0,0 +1,268 @@
+/**
+ * Unfortunately, Google is not providing the client for uploading files
+ * and we didn't found cool library for this feature
+ *
+ * We found it:
```

Comment:
>we didn't found cool library for this feature
>We found it:

So did we find it (the cool library) or we didn't find it? :) I'm confused.

Reply:
it's not cool library :) but it's something. and something is better than nothing :)